doc/source/feature_overview.md

```diff
@@ -0,0 +1,263 @@
+
+```{code-cell} ipython3
+# Plot proximity to vector
+rast_proximity_to_vec = vect.proximity(rast)
+rast_proximity_to_vec.show(cbar_title="Distance to glacier outline")
+vect.show(rast_proximity_to_vec, fc="none")
+```
+
+```{tip}
+To quickly visualize a raster directly from a terminal, without opening a Python console/notebook, check out our tool `geoviewer.py` in the {ref}`cli` documentation.
+```
+
+## Pythonic arithmetic and NumPy interface
+
+All {class}`~geoutils.Raster` objects support Python arithmetic ({func}`+<operator.add>`, {func}`-<operator.sub>`, {func}`/<operator.truediv>`, {func}`//<operator.floordiv>`, {func}`*<operator.mul>`,
+{func}`**<operator.pow>`, {func}`%<operator.mod>`) with any other {class}`~geoutils.Raster`, {class}`~numpy.ndarray` or
+number. With another {class}`~geoutils.Raster`, the georeferencing must match, while only the shape with a {class}`~numpy.ndarray`.
+
+```{code-cell} ipython3
+# Add 1 to the raster array
+rast += 1
+```
+
+Additionally, the class {class}`~geoutils.Raster` possesses a NumPy masked-array interface that allows to apply to it any [NumPy universal function](https://numpy.org/doc/stable/reference/ufuncs.html) and
+most other NumPy array functions, while logically casting {class}`dtypes<numpy.dtype>` and respecting {attr}`~geoutils.Raster.nodata` values.
+
+```{code-cell} ipython3
+# Apply a normalization to the raster
+import numpy as np
+rast = (rast - np.min(rast)) / (np.max(rast) - np.min(rast))
+```
+
+## Casting to {class}`~geoutils.Mask`, indexing and overload
+
+All {class}`~geoutils.Raster` classes also support Python logical comparison operators ({func}`==<operator.eq>`, {func}` != <operator.ne>`, {func}`>=<operator.ge>`, {func}`><operator.gt>`, {func}`<=<operator.le>`,
+{func}`<<operator.lt>`), or more complex NumPy logical functions. Those operations automatically casts them into a {class}`~geoutils.Mask`, a subclass of {class}`~geoutils.Raster`.
+
+```{code-cell} ipython3
+# Get mask of an AOI: infrared index above 0.7, at least 200 m from glaciers
+mask_aoi = np.logical_and(rast > 0.7, rast_proximity_to_vec > 200)
+```
+
+Masks can then be used for indexing a {class}`~geoutils.Raster`, which returns a {class}`~numpy.ma.MaskedArray` of indexed values.
+
+```{code-cell} ipython3
+# Index raster with mask to extract a 1-D array
+values_aoi = rast[mask_aoi]
+```
+
+Masks also have simplified, overloaded {class}`~geoutils.Raster` methods due to their boolean {class}`dtypes<numpy.dtype>`. Using {func}`~geoutils.Raster.polygonize` with a
+{class}`~geoutils.Mask` is straightforward, for instance, to retrieve a {class}`~geoutils.Vector` of the area-of-interest:
+
+```{code-cell} ipython3
+# Polygonize areas where mask is True
+vect_aoi = mask_aoi.polygonize()
+```
+
+```{code-cell} ipython3
+# Plot result
+rast.show(cmap='Reds', cbar_title='Normalized infrared')
+vect_aoi.show(fc='none', ec='k', lw=0.5)
```

Comment:
The polygons are really difficult to see as they are quite small... Either use a different color/linewidth or use a different threshold?

Reply:
Changed both linewidth and threshold.